samples/samples-powershell/AddProductWithIdentityColumnIncluded/run.ps1

```diff
@@ -1,16 +1,16 @@
 using namespace System.Net
 
 # Trigger binding data passed in via param block
-param($Request)
+param($Request, $TriggerMetadata)
 
 # Write to the Azure Functions log stream.
 Write-Host "PowerShell function with SQL Output Binding processed a request."
 
 # Update req_query with the body of the request
 $req_query = @{
-    productId= if($Request.QUERY.productId) { $Request.QUERY.productId } else { $null };
+    productId= if($Request.QUERY.productId) { [int]$Request.QUERY.productId } else { $null };
```

Comment:
Why are you casting these to ints? Your description doesn't say anything about these changes.

Reply:
Updated description to address this.

Reply:
So just to be clear - this isn't actually necessary? You're just doing it to be consistent?

Reply:
Yes, this is to be consistent with testing. These tests are currently not being run due to them being unstable. For now, I would like to have it be exactly the same as C# in order to reduce the variables of why it was not working.